cmd/client/main.go

```diff
@@ -58,7 +58,7 @@ func run(raddr string, p string) error {
 	if _, err := str.Write([]byte(msg)); err != nil {
 		return err
 	}
-	if err := str.Close(); err != nil {
+	if err := str.CloseWrite(); err != nil {
```

Comment:
We still need to call close eventually (e.g., in a defer).

Reply:
We could, but streams are automatically closed when the connection is closed, so this is not necessary.

Reply:
On the other hand, this is an example, and we should probably do things the clean way. Will add the defers.